systemtest/src/test/java/io/strimzi/systemtest/recovery/RecoveryST.java

```diff
@@ -2,10 +2,11 @@
  * Copyright Strimzi authors.
  * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
  */
-package io.strimzi.systemtest;
+package io.strimzi.systemtest.recovery;
```

Comment:
Surely this is part of the `operators` package

Reply:
So maybe both classes from `recovery` put into `operators.cluster` ?